examples/ClientExample.py

```diff
@@ -73,6 +73,8 @@ def main_test():
                 (threading.current_thread().getName(), space_name))
         # Get one client
         client = GraphClient(connection_pool)
+        # when connection is broken use it to as current session's space
```

Comment:
when connection is broken use the following method as the current session's space